tests/prague/eip6206_jumpf/contracts.py

```diff
@@ -0,0 +1,244 @@
+"""
+EOF V1 Code Validation tests
+"""
+
+from typing import List
+
+from ethereum_test_tools import EOFException
+from ethereum_test_tools.eof.v1 import Container, Section
+from ethereum_test_tools.eof.v1.constants import NON_RETURNING_SECTION
+from ethereum_test_tools.vm.opcode import Opcodes as Op
+
+
+def quick_code(code, inputs=0, outputs=NON_RETURNING_SECTION, height=0):
+    """A sorter way to write code with section 0 defaults"""
+    return Section.Code(
+        code=code, code_inputs=inputs, code_outputs=outputs, max_stack_height=height
+    )
+
+
+def container_name(c: Container):
+    """
+    Return the name of the container for use in pytest ids.
+    """
+    if hasattr(c, "name"):
+        return c.name
+    else:
+        return c.__class__.__name__
+
+
+def generate_jumpf_target_rules():
+    """
+    Generate tests for JUMPF where we are testing the validity of the JUNMPF target.
+    We are not testing stack so a lot of the logic is to get correct stack values.
+    """
+    valid = []
+    invalid = []
+    for current_outputs in [NON_RETURNING_SECTION, 0, 2, 4]:
```

Comment:
can you use pytest parametrize for this variable?

Reply:
Re-written as parameterized.  This basically makes this PR a do-over review wise.